### PR TITLE
LPS-128915 Make sure to always set the location whenever needed while taking advantage of only initializing one map

### PR DESCRIPTION
**Pull request by rodrigopaulino**
Related ticket: https://issues.liferay.com/browse/LPS-128915

### CODE REVIEW

**Comment by rodrigopaulino** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/constants/DDMFormRendererConstants.java [hunk 1]
```diff
@@ -21,6 +21,8 @@
  */
 public class DDMFormRendererConstants {
 
+	public static final String DDM_FORM_FIELD_EDITED_SUFFIX = "_edited";
```
Comment (jeyvison) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/constants/DDMFormRendererConstants.java:24]: This is being used in only one place, can we just use it directly?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-values-factory/src/main/java/com/liferay/dynamic/data/mapping/form/values/factory/internal/DDMFormValuesFactoryImpl.java [hunk 2]
```diff
@@ -371,23 +370,15 @@ protected String getDDMFormFieldParameterPrefix(
...
-
-		StringBundler sb = new StringBundler(4);
-
-		sb.append(DDMFormRendererConstants.DDM_FORM_FIELD_NAME_PREFIX);
-		sb.append(ddmFormFieldParameterName);
-		sb.append(
-			DDMFormRendererConstants.DDM_FORM_FIELD_LANGUAGE_ID_SEPARATOR);
-		sb.append(LocaleUtil.toLanguageId(locale));
+		String fullDDMFormFieldParameterName,
+		String defaultDDMFormFieldParameterValue) {
```
Comment (jeyvison) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-values-factory/src/main/java/com/liferay/dynamic/data/mapping/form/values/factory/internal/DDMFormValuesFactoryImpl.java:374]: Sort

**Review by jeyvison — CHANGES_REQUESTED** [review]
Hi @rodrigopaulino , can you please check my comments and the rebaase problem? Thanks!

**Comment by rodrigopaulino** [comment]
Resent https://github.com/liferay-data-engine/liferay-portal/pull/352

**Pull request closed without merging**